src/prefect/deployments.py

```diff
@@ -362,6 +364,9 @@ def _yaml_dict(self) -> dict:
         description="One of more tags to apply to this deployment.",
     )
     schedule: schemas.schedules.SCHEDULE_TYPES = None
+    is_schedule_active: Optional[bool] = Field(
+        default=True, description="Whether or not the schedule is active."
```

Comment:
I think we may want to default to `None` to respect the server-side value during updates.

I'm tempted to suggest another name for this as well, `schedule_active` seems nicer for this user-facing toggle whereas `is_schedule_active` makes more sense on the API object itself.

Reply:
cc @billpalombi

Reply:
>I think we may want to default to None to respect the server-side value during updates.
Yep, good point!